playgrounds/next/package.json

```diff
@@ -14,7 +14,7 @@
     "@types/node": "*",
     "@types/react": "^18",
     "@types/react-dom": "^18",
-    "next": "^14.2.3",
+    "next": "^14.2.10",
```

Comment:
I noticed the latest version for `next` is 14.2.13. Should we upgrade this as well?